src/src_V2/modules/global/hooks/use-filters.js

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
Also i have created the modules folder for serve as a module, i mean in this folder can be for example manager/superAdmin/Auth modules and the global folder which you have created is kinda some helper or util i think it'll be better to puth this global folder to src level in this case src_V2.